clients/iceberg-views/src/main/java/org/apache/iceberg/view/VersionParser.java

```diff
@@ -106,14 +106,19 @@ static Version fromJson(JsonNode node) {
       Iterator<String> fields = summaryNode.fieldNames();
       while (fields.hasNext()) {
         String field = fields.next();
-        if (field.equals(OPERATION)) {
-          builder.put(OPERATION, JsonUtil.getString(field, summaryNode));
-        } else if (field.equals(CommonViewConstants.GENIE_ID)) {
-          builder.put(CommonViewConstants.GENIE_ID, JsonUtil.getString(field, summaryNode));
-        } else if (field.equals(CommonViewConstants.ENGINE_VERSION)) {
-          builder.put(CommonViewConstants.ENGINE_VERSION, JsonUtil.getString(field, summaryNode));
-        } else {
-          builder.put(field, JsonUtil.getString(field, summaryNode));
+        switch (field) {
+          case OPERATION:
+            builder.put(OPERATION, JsonUtil.getString(field, summaryNode));
+            break;
+          case CommonViewConstants.GENIE_ID:
+            builder.put(CommonViewConstants.GENIE_ID, JsonUtil.getString(field, summaryNode));
+            break;
+          case CommonViewConstants.ENGINE_VERSION:
+            builder.put(CommonViewConstants.ENGINE_VERSION, JsonUtil.getString(field, summaryNode));
+            break;
+          default:
+            builder.put(field, JsonUtil.getString(field, summaryNode));
+            break;
```

Comment:
arent all switch cases doing the exact same thing? :thinking: 
```
builder.put(field, JsonUtil.getString(field, summaryNode));
```

Reply:
Duh - did not want to touch iceberg-views at all.
Reverted this change.

@nastra there are a couple more issues, some are actual issues. But iceberg-views is WIP stuff - so IDK what to do with the module.